.github/workflows/pull_request_closed.yml

```diff
@@ -11,6 +11,21 @@ env:
   SANITY_AUTH_TOKEN: ${{ secrets.SANITY_AUTH_TOKEN }}
 
 jobs:
+  deploy-studio-graphql:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+
+      - name: Deploy Studio Graphql Server
+        run: |
+          npm ci
+          npx sanity graphql deploy --no-playground --dataset development
```

Comment:
Should that be the production data set?

Reply:
Before deploy previews the development set graphql API was deployed in the pull request action. 

This is a way to keep the development api in sync with production. This way you can keep your branches up to date with the main branch and still develop against the development API. There are also graphql API deploys based on the tags so you can target that when working on a PR.

I'm on mobile bet I'll give examples of that later

Reply:
For instance this pr is deploy in the [`pull_request` action](https://github.com/cdierkens/bluebird-cabinetry-and-design/runs/2318607902?check_suite_focus=true):
```
Dataset: development
Tag: pull-request-809

GraphQL API deployed to:
https://5sj6awst.api.sanity.io/v1/graphql/development/pull-request-809
```

Here's a diagram of how I think about what sanity graphql APIs get deployed
![bluebird-sanity-graphql-deploys](https://user-images.githubusercontent.com/4812409/114319701-8cc4ed80-9ae0-11eb-9ba2-2c1358daac01.png)